packages/venia-ui/lib/components/CartPage/ProductListing/EditModal/editModal.js

```diff
@@ -29,45 +24,21 @@ import ProductForm from './productForm';
  * import EditModal from "@magento/venia-ui/lib/components/CartPage/ProductListing/EditModal";
  */
 const EditModal = props => {
-    const { item, setIsCartUpdating } = props;
+    const { item, setActiveEditItem, setIsCartUpdating } = props;
     const talonProps = useEditModal();
-    const { handleClose, isOpen, setVariantPrice, variantPrice } = talonProps;
+    const { setVariantPrice, variantPrice } = talonProps;
 
-    const classes = mergeClasses(defaultClasses, props.classes);
-    const rootClass = isOpen ? classes.root_open : classes.root;
-
-    const bodyComponent = item ? (
-        <div className={classes.body} key={item.id}>
-            <ProductDetail item={item} variantPrice={variantPrice} />
+    return item ? (
+        <Portal>
```

Comment:
I think we have duplicate `Portal`s going on here:

This `Portal` > `ProductForm` > `Dialog` > `Portal`.

I think we can remove this `Portal` safely.

Reply:
Yes, I removed this Portal.